MediaBundle/Resources/public/block/Carrousel/js/loadJs.js

```diff
@@ -0,0 +1,83 @@
+requirejs(
+    [
+     '../../../libs/jquery-2.0.2.min',
+     '../../../libs/jssor/jssor.core',
+     '../../../libs/jssor/jssor.slider',
+     '../../../libs/jssor/jssor.utils'
+    ],
+    function () {
+
+        $(document).ready(function(){
+/*            var _SlideshowTransitions = [
```

Comment:
@nicolasThal : In this PR, I only moved code from one place to another. The only code I modified was the namespaces. It's a requirement to make the carrousel working.

I assume this has been commented earlier because it was broking the page using this block.
Anyway the second step is to rewrite the rendering, I think I will discard all that code and the jssor library to use the fancybox library as in the gallery block.